NextGenConnect/JSON to HL7v2.xml

```diff
@@ -83,35 +83,257 @@ for (var i = 0; i &lt; getArrayOrXmlLength(patient[&apos;identifier&apos;]); i++
 }
 tmp[&apos;PID&apos;][&apos;PID.3&apos;] = ids.join(&apos;~&apos;);
 
-
-/*
 //PID-5 patient name
-tmp[&apos;PID&apos;][&apos;PID.5&apos;][&apos;PID.5.1&apos;] = patient[&apos;name&apos;][0][&apos;family&apos;][&apos;@value&apos;].toString();
-tmp[&apos;PID&apos;][&apos;PID.5&apos;][&apos;PID.5.2&apos;] = patient[&apos;name&apos;][0][&apos;given&apos;][&apos;@value&apos;].toString();
-tmp[&apos;PID&apos;][&apos;PID.5&apos;][&apos;PID.5.3&apos;] = &quot;&quot;; //not in XML
+tmp[&apos;PID&apos;][&apos;PID.5&apos;][&apos;PID.5.1&apos;] = patient[&apos;name&apos;][0][&apos;family&apos;];
+tmp[&apos;PID&apos;][&apos;PID.5&apos;][&apos;PID.5.2&apos;] = patient[&apos;name&apos;][0][&apos;given&apos;][0]; //Why is this stored in an array in sample JSON message?
+tmp[&apos;PID&apos;][&apos;PID.5&apos;][&apos;PID.5.7&apos;] = &quot;L&quot;;  // Legal Name		
+
+//PID-6 Mothers Maiden Name -- ??? Try to grab value ?
```

Comment:
If we aren't collecting this in the registration app, we can leave this out. Check with front end.

Reply:
Deleted.